src/commands/sarif/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const getBaseIntakeUrl = () => {
+  if (process.env.DATADOG_SITE || process.env.DD_SITE) {
+    return `https://cicodescan-intake.${process.env.DATADOG_SITE || process.env.DD_SITE}`
```

Comment:
is this intake ready?

Reply:
The intake should be ready for only-internal usage